lms/djangoapps/certificates/docs/decisions/001-allowlist-cert-requirements.rst

```diff
@@ -31,7 +31,7 @@ the time the certificate is generated:
   * The enrollment mode must be eligible for a certificate
   * The enrollment does not need to be active
 
-* The user must have an approved, unexpired, ID verification
 * The user must not have an invalidated certificate for the course run (see the *CertificateInvalidation* model)
 * HTML (web) certificates must be globally enabled, and also enabled for the course run
 * The user must be on the allowlist for the course run (see the *CertificateAllowlist* model)
+* The user is no longer required to have an approved, unexpired, ID verification, however, this can be toggled by enabling the `ENABLE_CERTIFICATES_IDV_REQUIREMENT` WaffleFlag
```

Comment:
Nit: As this document is trying to document what _is_ required, I would reword it to something like...
```
If the `ENABLE_CERTIFICATES_IDV_REQUIREMENT` WaffleFlag is enabled, a user must have an approved and unexpired ID verification.
```